packages/chain-adapters/src/utxo/UTXOBaseAdapter.ts

```diff
@@ -0,0 +1,159 @@
+import { CAIP2, caip2, caip19 } from '@shapeshiftoss/caip'
+import { bip32ToAddressNList, HDWallet, PublicKey } from '@shapeshiftoss/hdwallet-core'
+import { BIP44Params, chainAdapters, ChainTypes, UtxoAccountType } from '@shapeshiftoss/types'
+import { bitcoin } from '@shapeshiftoss/unchained-client'
+import WAValidator from 'multicoin-address-validator'
+
+import { ChainAdapter as IChainAdapter } from '../api'
+import { ErrorHandler } from '../error/ErrorHandler'
+import {
+  accountTypeToScriptType,
+  bnOrZero,
+  convertXpubVersion,
+  toRootDerivationPath
+} from '../utils'
+
+export type UTXOChainTypes = ChainTypes.Bitcoin // to be extended in the future to include other UTXOs
+
+export interface ChainAdapterArgs {
+  providers: {
+    http: bitcoin.api.V1Api //?
+    ws: bitcoin.ws.Client
+  }
+  coinName: string
+  chainId?: CAIP2
+}
+
+/**
+ * Base chain adapter for all UTXO chains. When extending please add your ChainType to the
+ * UTXOChainTypes. For example:
+ *
+ * `export type UTXOChainTypes = ChainTypes.Bitcoin | ChainTypes.Litecoin`
+ */
+export abstract class UTXOBaseAdapter<T extends UTXOChainTypes> implements IChainAdapter<T> {
+  protected chainId: CAIP2
+  protected coinName: string
+  protected readonly providers: {
+    http: bitcoin.api.V1Api
+    ws: bitcoin.ws.Client
+  }
+
+  public static readonly defaultBIP44Params: BIP44Params = {
+    purpose: 84, // segwit native
```

Comment:
also wanted to check on this, due to lack of experience with UTXO's and the `purpose` here.  Not sure if this should be moved up a level to the implementing class

Reply:
yeah - these BIP44Params should be moved into the implementing class imo. @cjthompson can weigh in here too